css/templates/lune/components/_header.css

```diff
@@ -0,0 +1,53 @@
+@import "../page-vars";
+
+.ampstart-headerbar {
+  justify-content: space-between
+
+  height: var(--layout-header-height-mobile);
+
+  /* Override ampstart default box shadow styles */
+  box-shadow: none;
+
+  /* header is fixed so should be moved onto a new GPU layer */
+  will-change: transform;
+}
+
+.ampstart-headerbar-home-link {
+  position: absolute;
```

Comment:
position absolute will make it not very usable/editable - you can tweak the margins in the other approach to make it have equal spacing on the side to avoid position absolute? 

Generally creating a new layers comes with certain downsides, lets try to avoid those where we can